app/api/search/search.js

```diff
@@ -447,6 +448,10 @@ const instanceSearch = elasticIndex => ({
         documentsQuery.limit(query.limit);
       }
 
+      if (query.offset) {
+        documentsQuery.offset(query.offset);
```

Comment:
I can't seem to find any test covering an 'offset' scenario.  I am missing something?

Reply:
Removed offset method since the query builder already had a method using the key "from" that is tested.